streampark-console/streampark-console-service/src/main/resources/application.yml

```diff
@@ -91,6 +91,8 @@ streampark:
   workspace:
     local: /opt/streampark_workspace
     remote: hdfs://hdfscluster/streampark   # support hdfs:///streampark/ 、 /streampark 、hdfs://host:ip/streampark/
+  archive:
+    path: hdfs:///streampark/completed-jobs/
```

Comment:
The archive path shouldn't be configured. It should under the `workspace.remote`. What do you think?

Reply:
write together, if it is oss, this remote seems to support only hdfs, but we are supporting a variety of storage media

Reply:
If so, we shouldn't use a new directory every time we need to use remote storage. A reasonable solution is : `workspace.remote` should support various storages, and `archive.path` should under the `workspace.remote`.

In summary: we should use `workspace.remote` uniformly, and in the future, do lifecycle management for all directories under `workspace.local` and workspace.remote`.

Reply:
I'd like to leave this pr unchanged by default for now, and modify it later in a separate pr. Because I'm not sure if my changes here will lead to other problems